envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -377,6 +378,22 @@ class DynamicForwardProxyProperties {
     var connectionTimeout = Duration.ofSeconds(1)
 }
 
+class CompressionProperties {
+    var gzip = GzipProperties()
+    var brotli = BrotliProperties()
+    var minContentLength = 100
+    var disableOnEtagHeader = true
+}
+
+class BrotliProperties {
+    var enabled = false
+    var quality = 11
```

Comment:
why do we use 11? did we check how much does the compression and time differs between different levels? or do we not care about additional latency and just want to compress as much as possible?

Reply:
I see that it's based on the Ziółek configuration - I guess they tested it and it's okay?

Reply:
so that's just a default configuration, rn I applied what Ziolek suggested, but I'm going to test it on dev to see how it affects cpu and latency

Reply:
ah, I thought it was hardcoded at the moment, forgot it's in properties, I guess it can be left that way and adjusted through configs